src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -732,10 +839,12 @@ def run(self):
                     subprocess.Popen(cmd, shell=True).communicate()
                 else:
                     telemetry.start()
+                    self.recommender.update_executing(cmd)
                     self.cli_execute(cmd)
                     if self.last_exit and self.last_exit != 0:
                         telemetry.set_failure()
                     else:
                         telemetry.set_success()
+                    self.recommender.update_exec_result(self.last_exit, telemetry.get_error_info()['result_summary'])
                     telemetry.flush()
```

Comment:
Please add corresponding comment for the three new steps

Reply:
Done